docs/_data/versions.yml

```diff
@@ -12,3 +12,5 @@
   stability: latest stable
 - version: 5.3.0
   stability: unstable
```

Comment:
Should this be changed to "stable"?

Reply:
> Should this be changed to "stable"?

Not yet.  Once we have a final build that passes all testing then we can declare it stable and at that time it will be changed.